pkg/ui/react-app/src/pages/targets/__testdata__/testdata.ts

```diff
@@ -1,5 +1,3 @@
-/* eslint @typescript-eslint/camelcase: 0 */
```

Comment:
Let's make this `/* eslint-disable camelcase */` and make this rule "warn" for all other files in eslintrc as said in the previous comment.